modules/site-navigation/module.php

```diff
@@ -44,4 +57,25 @@ public static function is_active() {
 
 		return Plugin::$instance->experiments->is_feature_active( 'editor_v2' );
 	}
+
+	/**
+	 * Register Experiment
+	 *
+	 * @since 3.16.0
+	 *
+	 * @return void
+	 * @throws \Exception
+	 */
+	private static function register_pages_panel_experiment() {
+		Plugin::$instance->experiments->add_feature( [
+			'name' => self::PAGES_PANEL_EXPERIMENT_NAME,
+			'title' => esc_html__( 'Pages Panel', 'elementor' ),
+			'release_status' => Experiments_Manager::RELEASE_STATUS_BETA,
```

Comment:
Please consult with Aviv about other settings of the exp
```suggestion
			'release_status' => Experiments_Manager::RELEASE_STATUS_ALPHA,
```